instrumentserver/server/core.py

```diff
@@ -233,7 +233,10 @@ def startServer(self) -> None:
             send(socket, response_to_client)
 
             self.messageReceived.emit(str(message), response_log)
-
+        # Importing at the top of the file causes a circular import
+        from ..apps import _quitPollingThread
+        _quitPollingThread()
```

Comment:
This should be removed right? it is not needed anymore now that we are passing the thread as an argument?